subprojects/core/src/integTest/groovy/org/gradle/api/tasks/TaskTimeoutIntegrationTest.groovy

```diff
@@ -268,15 +270,20 @@ class TaskTimeoutIntegrationTest extends AbstractIntegrationSpec {
         fails "block"
 
         and:
-        def logging = taskLogging(":block")
+        def outputLines = result.groupedOutput.task(":block").output.readLines()
+        outputLines[0] == "Requesting stop of task ':block' as it has exceeded its configured timeout of 500ms."
+        outputLines[1] == "Timed out task ':block' has not yet stopped."
+        outputLines[outputLines.size() - 1] == "Timed out task ':block' has stopped."
 
+        and:
+        def logging = taskLogging(":block")
```

Comment:
Why do you do both assertions now, the one using the build operation fixture and `result.groupedOutput`?

Reply:
I don't think they are the same thing.

The operations test test exactly which operation they are associated with. IIUC, that's not what the output test is doing. It's just testing how it is rendered, which encompasses child operations.